### PR TITLE
Update Lab #5

### PR DESCRIPTION
**Pull request by borjasotomayor**
Just made some minor updates; there didn't seem to be much questions about this lab on Piazza last year.

### CODE REVIEW

File: website/content/labs/lab5.md [hunk 1]
```diff
@@ -110,13 +99,17 @@ Create a new file in the `tests/` directory called `test_segment.c`. You’ll ne
 ```
 
 {{% warning %}}
+{{% md %}}
 **Warning**: From now on, you must do all of your work over `ssh`, or in CSIL, or using the [department’s virtual machine](https://howto.cs.uchicago.edu/vm:index). Each of those environments have the `criterion` library properly set up and ready to go. While it is possible to install `criterion` on an unsupported machine, we may not be able to provide support for that setup.
```
Comment (mpcsh) [inline website/content/labs/lab5.md]: We should make our first suggestion here the headless VM

File: website/content/labs/lab5.md [hunk 2]
```diff
@@ -110,13 +99,17 @@ Create a new file in the `tests/` directory called `test_segment.c`. You’ll ne
...
+{{% /md %}}
 {{% /warning %}}
 
 You will also need to modify the `Makefile` in the `tests/` directory to add your new file.
 
 {{% warning %}}
+{{% md %}}
 **Caution**: There are *two* `Makefile`s in this project! There’s the root-level `Makefile` for building the library, but then there’s also a `Makefile` in the `tests` directory specifically for building test files. You should only modify the latter.
```
Comment (mpcsh) [inline website/content/labs/lab5.md]: While it doesn't really matter, this is actually technically wrong - there are three (there's one in `samples/`)

File: website/content/labs/lab5.md (hunk 1 above)
Comment (borjasotomayor) [reply website/content/labs/lab5.md]: Fixed

File: website/content/labs/lab5.md (hunk 2 above)
Comment (borjasotomayor) [reply website/content/labs/lab5.md]: Fixed

**Pull request merged**